flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/nodes/CommonCalc.scala

```diff
@@ -52,10 +53,18 @@ trait CommonCalc {
       rowType.getFieldNames,
       expandedExpressions)
 
+    val retractionProcess =
```

Comment:
rename retractionProcess to retractionProcessCode
or moving this part to CodeGenerator considering the consistency of code style.
And I prefer the later one.